pages/team.tsx

```diff
@@ -0,0 +1,31 @@
+import type { AppProps } from 'next/app';
+import { Flex, Container, Heading, Stack, Text } from '@chakra-ui/react';
+import Layout from '@packages/components/Layout';
+import { ChakraProvider, CSSReset } from '@chakra-ui/react';
+import Metadata from '@packages/components/Metadata';
+import ModalProvider from '@packages/features/modal-context';
+
+function MyApp() {
+  return (
+    <Container maxW={'5xl'}>
+      <Stack
+        textAlign={'center'}
+        align={'center'}
+        spacing={{ base: 8, md: 10 }}
+        py={{ base: 20, md: 28 }}
+      >
+        <Heading
+          fontWeight={600}
+          fontSize={{ base: '3xl', sm: '4xl', md: '4xl' }}
+          lineHeight={'110%'}
+        >
+          Em breve apresentaremos nossos membros.{' '}
+          <Text as={'span'} color={'purple.400'}>
+            Aguarde!!!
+          </Text>
+        </Heading>
+      </Stack>
+    </Container>
+  );
+}
+export default MyApp;
```

Comment:
Que tal mover o `export default` pra frente da declaração do componente?

É meio frustrante chegar no final do arquivo pra saber quem está sendo exportado. 
PS: Sei que eu fiz isso em alguns lugares, mas eu to errado, hahaha